src/library/process.js

```diff
@@ -98,6 +127,7 @@ export const processMessage = async (AWS, region, service, account,
         ...currAccData.vendorData[`${service}`],
         ...procRes.serviceUserData,
       };
+      // CR: Mickey: why use batchPut here knowing it will only ever be one value instead of making a single put helper?
```

Comment:
@MickeyMcc Used it because it is a direct pull from my earlier code that was thoroughly tested and working for a long time. No other reason.